src/paperless/auto_auth.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.models import User
```

Comment:
I'm not sure why you're importing `User` here if you're going to redefine it below.